src/components/ProjectListItem.tsx

```diff
@@ -1,7 +1,21 @@
 import React, { FunctionComponent } from "react";
 import { CalendarUIProps, CalendarAction } from "../calendar/types";
-import { ListItem, ListItemText, Checkbox } from "@material-ui/core";
+import {
+  ListItem,
+  ListItemText,
+  Checkbox,
+  makeStyles,
+  IconButton,
+} from "@material-ui/core";
 import Project from "../calendar/Project";
+import InfoIcon from "@material-ui/icons/Info";
+import Divider from "@material-ui/core/Divider";
```

Comment:
when these material/core imports come up, put them in the import list that's already happening above